File: Framework/Domain/Dtos/Common/DownloadFileInfo.py

from pydantic import BaseModel
from typing import Optional

class DownloadFileInfo(BaseModel):
    """
    Pydantic model equivalent to the C# DownloadFileInfo DTO.
    Represents file information for download.
    """
    file_path: str
    content_type: str
    file_name: str
    # Add other properties if they exist in the full C# file

File: Framework.Domain/Dtos/Common/DownloadFileInfo.cs
namespace xCodie.Application.DTOs.Commom;

public class DownloadFileInfo(string filePath, string contentType, string fileName)
{
    public string FilePath { get; set; } = filePath;
    public string ContentType { get; set; } = contentType;
    public string FileName { get; set; } = fileName;
}
